src/quo/components/wallet/account_card/schema.cljs

```diff
@@ -0,0 +1,33 @@
+(ns quo.components.wallet.account-card.schema)
+
+(def ?base
+  [:map
+   [:type {:optional true} [:enum :default :watch-only :add-account :empty :missing-keypair]]
+   [:customization-color {:optional true} [:maybe :schema.common/customization-color]]
+   [:theme :schema.common/theme]
+   [:metrics? {:optional true} [:maybe :boolean]]
+   [:on-press {:optional true} [:maybe fn?]]])
+
+(def ?amount
```

Comment:
I think we should mark internal schemas private

Reply:
I like how you split up the shcemas 🙌

Reply:
Done